recipes/impact-t/meta.yaml

```
@@ -0,0 +1,79 @@
+{% set name = "impact-t" %}
+{% set version = "2.0" %}
+
+{% set build = 1000 %}
```

This should start from 0:
```suggestion
{% set build = 0 %}
```

---

Addressed via https://github.com/conda-forge/staged-recipes/pull/13701/commits/49c05e7cf6304bd40867832a6f5fe339a461a6fa

I followed the pattern at some recipes in which the number start at 1000 to favor the non-mpi build.
I switched it back to 0.

---

Yeah that's a legacy thing: https://conda-forge.org/docs/maintainer/maintainer_faq.html#mfaq-build-number-1000

I've opened a PR to update the docs: https://github.com/conda-forge/conda-forge.github.io/pull/1246